### ISSUE TITLE
Add option to support client certificates

### ISSUE THREAD

**Issue by rgl** [jakubroztocil/httpie#163]
Add the --cert CERT_FILE option to support client certificates.

This was quite straightforward to add because the requests library already supports this. I did a quick patch at:

https://dl.dropboxusercontent.com/u/20421111/add-support-for-client-certificates.patch

Maybe we should check whether the file exists before passing the control to requests?

**Comment by rbarlow** [comment]
The Readme.md says that --cert and --certkey are supported, but it appears that --cert is not recognized with the latest version available from PyPI:

```
$ http --verify=no --cert ~/.pulp/user-cert.pem GET https://example.com/pulp/api/v2/content/units/rpm/
usage: http [--json] [--form] [--pretty {all,colors,format,none}]
            [--style STYLE] [--print WHAT] [--verbose] [--headers] [--body]
            [--stream] [--output FILE] [--download] [--continue]
            [--session SESSION_NAME_OR_PATH | --session-read-only SESSION_NAME_OR_PATH]
            [--auth USER[:PASS]] [--auth-type {basic,digest}]
            [--proxy PROTOCOL:PROXY_URL] [--follow] [--verify VERIFY]
            [--timeout SECONDS] [--check-status] [--ignore-stdin] [--help]
            [--version] [--traceback] [--debug]
            [METHOD] URL [REQUEST_ITEM [REQUEST_ITEM ...]]
http: error: unrecognized arguments: --cert
```

**Comment by jakubroztocil** [comment]
This is available since [0.9.0](https://github.com/jakubroztocil/httpie/blob/master/CHANGELOG.rst#090-2015-01-31):

> Added `--cert` and `--cert-key` parameters to specify a client side certificate and private key for SSL

**Comment by arivasvera** [comment]
cert phrase/password is supported?
